app/main/views/brief_responses.py

```diff
@@ -223,6 +224,13 @@ def list_brief_responses():
     if brief_id is not None:
         brief_responses = brief_responses.filter(BriefResponse.brief_id == brief_id)
 
+    if awarded_at is not None:
+        day_start = datetime.strptime(awarded_at, "%Y-%m-%d")
```

Comment:
Do we not already have a standard date-argument-parsing function in the api? (the answer may well be no)

Reply:
Quite right! Have updated 👍